guardian/recipes-backend
# Ensure eslint configuration integrates prettier properly

## What does this change?

Amends eslint configuration in an attempt to integrate Prettier properly, so that linting is consistent for all developers:
- change `extends` to include `"prettier"`
- adds `settings` section to specify `import/resolver` settings
- ignores `jest.config.ts` when linting

Also see changes described in PR #66, which was merged into this PR

As a result of improved linting, it was also necessary to fix 412 linting errors!

## How to test

`npm run lint` - no linting errors shown
`npm run test` - tests pass
`npm run build` - no errors

I guess this should also be deployed to CODE?  I welcome advice on how best to test it once deployed

> Yes I'll deploy to CODE, publish a recipe, and check the GET endpoints. POST endpoints are pretty much deprecated now along with Mise-En-Place so don't need to worry about those so much

## How can we measure success?

No more PRs where it's difficult to see the actual changes because of all the linting changes

## Have we considered potential risks?

Greater consistency is surely a good thing!
